GOVERNANCE.md

```diff
@@ -36,10 +36,17 @@ Collaborators. All pull requests must be reviewed and accepted by a
 Collaborator with sufficient expertise who is able to take full
 responsibility for the change. In the case of pull requests proposed
 by an existing Collaborator, an additional Collaborator is required
-for sign-off. Consensus should be sought if additional Collaborators
-participate and there is disagreement around a particular
-modification. See [Consensus Seeking Process](#consensus-seeking-process) below
-for further detail on the consensus model used for governance.
+for sign-off.
+
+If there is disagreement among Collaborators about whether a proposed change
+should be accepted, then the change may not be accepted unless:
+
+* discussion and/or additional changes result in no Collaborators objecting to
+  the change; previously-objecting Collaborators do not necessarily have to
+  sign-off on the change, but they should not be opposed to it
+* the change is escalated to the CTC and the CTC approves the change; this
```

Comment:
I guess for something as significant as resolving an impasse on a controversial change, we should require a vote. I'll update the text to say that. We can always change the rules if that turns out to be onerous. But it seems that this comes up relatively infrequently.